app/css/style.css

```diff
@@ -75,4 +75,7 @@ address {
 .btn-cost-calculator {
     padding-left: 0.5rem;
     padding-right: 0.5rem;
+}
+.padding-4 {
```

Comment:
@strivenabil this is not necessary and pad practice if using Tailwind. You should be able to use a built in class such as py-10. See https://tailwindcss.com/docs/padding